### DISCUSSION TITLE
Is It Time For V To Have a Wikipedia Page?

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#17247] [General]
Is it time for V to have a Wikipedia page?

**Comment by JalonSolov** [comment]
Not until V can pass the "notability" test.

https://en.wikipedia.org/wiki/Wikipedia:Notability

**Reply by Wajinn** [reply]
> Not until V can pass the "notability" test.

It's up to team V, so when you guys are ready, then it will be so.  Plus there might be some strategy at play.  Just saying, seems like much lesser known languages have wikipedia pages, but V has done well without it and per GitHub stars.

**Reply by bogen85** [reply]
Show us one of the lesser known languages that has a Wikipedia page and did not not pass the "Notability" test.

**Reply by JalonSolov** [reply]
A Wikipedia page was created for V a while ago, and they took it down because it didn't pass the notability test.  Doesn't matter if some other language has a page or not.

**Reply by Wajinn** [reply]
> Show us one of the lesser known languages that has a Wikipedia page and did not not pass the "Notability" test.

> A Wikipedia page was created for V a while ago, and they took it down because it didn't pass the notability test. Doesn't matter if some other language has a page or not.


I was under the mistaken impression that team V didn't want to create a Wikipedia page (for unknown reasons), not that they were having issues with the Wikipedia "overlords".  It's more clear as to what's going on.

Also, surprised to not see a V page, when these languages managed to get pages: Boo, Joy, Pizza, Red, Ring, Crystal, Nim, Zig, Seed7, Reason, etc...  Particular situations can be quite different or difficult.  So, that's how it is for now.

**Reply by bogen85** [reply]
It would a appear that languages that don't have a sufficient level of popularity yet somehow slip under the radar of the notability police.

- I don't see how Boo can have an entry. (slipped under the radar?)
- Joy was written up in Dr. Dobbs and ACM had a few other papers written on it.
- Pizza ??? (slipped under the radar?)
- Red (slipped under the radar?)
- Ring I don't see.
- Crystal (seems weak, does have some write-ups on a couple websites other than the main one)
- Nim - (write-ups on many websites, also more than one book)
- Zig - (write-ups on many websites, I don't see any books listed, seems weak, slipped under the radar?)
- Seed7 - (write-ups, a book, at least one thesis)
- Reason -  (write-ups, seems weak, slipped under the radar?)

**Reply by Wajinn** [reply]
V actually has 2 books (that I know of) published and on Amazon.

[The V Programming Language basic (in Japanese and 1st one)](https://www.amazon.com/Programming-Language-basic-Japanese-ebook/dp/B08BKJDRFR)

[Getting Started with V Programming](https://www.amazon.com/Getting-Started-Programming-end-end-ebook/dp/B09FKK3JL7)

**Reply by bogen85** [reply]
> V actually has 2 books (that I know of) published and on Amazon.

With those as references (along with other writes ups on V that are not from the V author/team), and mentioned in an updated V Wikipedia article (that is written from a scientific non-biased approach, that does not come across as a fan boy entry) then the Wikipedia Notability Police should be satisfied.
